xtask/src/ci/target_matrix.rs

```diff
@@ -196,4 +259,34 @@ mod tests {
         let matrix = run(["--none"]);
         assert_eq!(&Vec::<CiTarget>::new(), &matrix);
     }
+
+    #[test]
+    fn prs() {
+        assert_eq!(process_bors_message("Merge #1337").unwrap().0, vec!["1337"]);
+        assert_eq!(
+            process_bors_message("Merge #1337 #42").unwrap().0,
+            vec!["1337", "42"]
+        );
+        assert_eq!(
+            // the trailing space is intentional
+            process_bors_message("Try #1337: ").unwrap().0,
+            vec!["1337"]
+        );
+    }
```

Comment:
ah, these are technically wrong

```suggestion
        assert_eq!(process_bors_message("Merge #1337\n").unwrap().0, vec!["1337"]);
        assert_eq!(
            process_bors_message("Merge #1337 #42\n").unwrap().0,
            vec!["1337", "42"]
        );
        assert_eq!(
            // the trailing space is intentional
            process_bors_message("Try #1337: \n").unwrap().0,
            vec!["1337"]
        );
    }
```